base/system/services/database.c

```diff
@@ -1395,15 +1500,15 @@ ScmControlService(HANDLE hControlPipe,
                 if (bResult == FALSE)
                 {
                     dwError = GetLastError();
-                    DPRINT1("GetOverlappedResult() failed (Error %lu)\n", dwError);
+                    DPRINT("(%S, %d) GetOverlappedResult() failed (Error %lu)\n", lpService->lpServiceName, dwControl, dwError);
```

Comment:
Just a question out of curiosity: Why the DPRINT1->DPRINT in this error-scenario? Did it spam?